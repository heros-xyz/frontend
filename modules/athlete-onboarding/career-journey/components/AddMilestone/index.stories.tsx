import type { Meta, StoryObj } from "@storybook/react";
import AddMilestone from ".";

const meta: Meta<typeof AddMilestone> = {
  title: "Athlete/AddMilestone",
  component: AddMilestone,
  tags: ["autodocs"],
  parameters: {
    layout: "base",
  },
};

export default meta;
type Story = StoryObj<typeof AddMilestone>;

export const AddMilestoneStory: Story = {};
