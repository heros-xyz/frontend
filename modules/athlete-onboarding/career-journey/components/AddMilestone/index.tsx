import {
  Box,
  Button,
  Center,
  Checkbox,
  Flex,
  Grid,
  GridItem,
  Stack,
  Text,
  Textarea,
} from "@chakra-ui/react";
import { useFormik } from "formik";
import { useState } from "react";
import { If, Then } from "react-if";
import DateSelect from "@/components/ui/DateSelect";
import ErrorMessage from "@/components/common/ErrorMessage";
import { SPORT_ICONS_MOCK } from "@/utils/mock-icon";
import { IMilestone, initialValues, validationSchema } from "../../constants";
import { styles } from "./styles";

interface IProp {
  onSubmit: (values: IMilestone) => void;
  values?: IMilestone;
}

const AddMilestone: React.FC<IProp> = ({ values, onSubmit }) => {
  const [iconCheck, setIconCheck] = useState("");

  const formik = useFormik({
    initialValues: values || initialValues,
    validationSchema,
    onSubmit: (values) => {
      onSubmit(values);
    },
  });

  const handleClickIcon = (value: string, iconCheck: string) => {
    setIconCheck(value !== iconCheck ? value : "");
    formik.setFieldValue("icon", value === iconCheck ? "" : value);
  };

  return (
    <Box
      w="auto"
      bg="secondary"
      minH="100vh"
      fontSize={{ base: "sm", xl: "xl" }}
      color="primary"
      alignItems="baseline"
    >
      <Center
        pt={{ base: 5, xl: "4rem" }}
        fontWeight="extrabold"
        fontSize={{ base: "xs", xl: "xl" }}
      >
        CAREER JOURNEY
      </Center>
      <Text mt={7} fontWeight="bold" fontSize={{ base: "md ", xl: "xl" }}>
        ADD MILESTONE
      </Text>
      <Box fontSize={{ base: "sm", lg: "xl" }}>
        <form onSubmit={formik.handleSubmit}>
          <Flex mt={4}>
            <Checkbox
              id="isPeriodDate"
              name="isPeriodDate"
              onChange={formik.handleChange}
              isChecked={formik.values.isPeriodDate}
              colorScheme="facebook"
              borderColor="#313F4C"
              sx={{
                span: {
                  borderRadius: 4,
                },
              }}
            />

            <Text
              ml="2.5"
              fontSize={{ base: "xs", lg: "md" }}
              fontWeight="medium"
            >
              I want this to be a period of time
            </Text>
          </Flex>
          <Box mb={{ base: 10, lg: 20 }} color="black.ish">
            <Box fontWeight="medium" mt={7}>
              {formik.values.isPeriodDate ? "Start" : "Enter"} Date
              <Text as="span" color="error.dark">
                {" "}
                *
              </Text>
            </Box>
            <Flex justifyContent="space-between" gap={{ base: 3, lg: 8 }}>
              <DateSelect
                date={formik.values?.startDate}
                onChange={(value) => formik.setFieldValue("startDate", value)}
                submitted={!!formik.submitCount}
                zIndex={20}
              />
            </Flex>
            <ErrorMessage
              mt={0.5}
              condition={Boolean(
                formik.submitCount && formik.errors?.startDate
              )}
              errorMessage={formik.errors?.startDate}
            />
          </Box>

          <If condition={formik.values.isPeriodDate}>
            <Then>
              <Box mb={{ base: 12, lg: 20 }} color="black.ish">
                <Box fontWeight="medium" mt={7}>
                  End Date
                  <Text as="span" color="error.dark">
                    {" "}
                    *
                  </Text>
                </Box>
                <Flex justifyContent="space-between" gap={{ base: 3, lg: 8 }}>
                  <DateSelect
                    date={formik.values?.endDate}
                    onChange={(value) => formik.setFieldValue("endDate", value)}
                    submitted={!!formik.submitCount}
                  />
                </Flex>
                <ErrorMessage
                  mt={0.5}
                  condition={Boolean(
                    formik.submitCount && formik.errors?.endDate
                  )}
                  errorMessage={formik.errors?.endDate}
                />
              </Box>
            </Then>
          </If>

          <Box mb={7}>
            <Box fontWeight="medium">
              Enter Name/Title
              <Text as="span" color="error.dark">
                {" "}
                *
              </Text>
            </Box>
            <Textarea
              id="title"
              name="title"
              variant="flushed"
              onChange={formik.handleChange}
              value={formik.values.title}
              placeholder="It can be name of Medal/Certificate/Prize or anything that important to you"
              resize={"none"}
              size={{ base: "sm", xl: "md" }}
              minH={16}
              style={styles.textarea}
              isInvalid={Boolean(formik.errors.title && formik.touched.title)}
            />
            <ErrorMessage
              mt={0.5}
              condition={formik.errors.title && formik.touched.title}
              errorMessage={formik.errors.title}
            />
          </Box>

          <Box mb={7}>
            <Box fontWeight="medium">Enter Description/Subtitle</Box>
            <Textarea
              id="description"
              name="description"
              variant="flushed"
              onChange={formik.handleChange}
              value={formik.values.description}
              placeholder="Describe further for your fan to understand (Where did it happen/ How...)"
              resize={"none"}
              size={{ base: "sm", xl: "md" }}
              minH={16}
              style={styles.textarea}
              isInvalid={Boolean(
                formik.errors.description && formik.touched.description
              )}
            />
            <ErrorMessage
              mb={8}
              condition={
                formik.errors.description && formik.touched.description
              }
              errorMessage={formik.errors.description}
            />
          </Box>

          <Box mb={8}>
            <Box fontWeight="medium" mb={3}>
              Choose an icon to represent the milestone
            </Box>
            <Box fontWeight="normal" fontSize={{ base: "xs", lg: "md" }} mb={4}>
              Make your milestone even more interesting!
            </Box>
            <Grid templateColumns="repeat(2, 1fr)" gap={1}>
              {SPORT_ICONS_MOCK.map((el) => {
                return (
                  <GridItem key={el.value} w="100%" pb="2">
                    <Flex>
                      <Button
                        w={{ base: "54px", xl: "80px" }}
                        h={{ base: "54px", xl: "80px" }}
                        borderRadius="full"
                        bg={el.value === iconCheck ? "primary" : "accent.4"}
                        color={el.value === iconCheck ? "secondary" : "primary"}
                        onClick={() => handleClickIcon(el.value, iconCheck)}
                        _hover={{}}
                        _active={{}}
                      >
                        {el.Icon}
                      </Button>
                      <Text
                        display="flex"
                        alignItems="center"
                        ml={1}
                        fontSize={{ base: "xs", xl: "md" }}
                      >
                        {el.name}
                      </Text>
                    </Flex>
                  </GridItem>
                );
              })}
            </Grid>
          </Box>
          <Button
            variant="primary"
            w={{ base: "100%", xl: "auto" }}
            mt={2}
            type="submit"
          >
            SAVE
          </Button>
        </form>
      </Box>
    </Box>
  );
};

export default AddMilestone;
