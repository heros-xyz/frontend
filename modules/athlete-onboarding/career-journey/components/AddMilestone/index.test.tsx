import { render } from "@testing-library/react";
import AddMilestone from ".";

it("renders homepage unchanged", () => {
  const onChangeStep = (step: number) => {
    console.log("test", step);
  };
  const { container } = render(
    <AddMilestone onSubmit={() => onChangeStep(2)} />
  );
  expect(container).toMatchSnapshot();
});
