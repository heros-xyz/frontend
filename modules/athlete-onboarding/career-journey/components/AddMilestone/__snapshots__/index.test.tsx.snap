// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-1d4zvd6"
  >
    <div
      class="css-17etjhj"
    >
      CAREER JOURNEY
    </div>
    <p
      class="chakra-text css-15wsqor"
    >
      ADD MILESTONE
    </p>
    <div
      class="css-z2k2e5"
    >
      <form>
        <div
          class="css-1n0zz00"
        >
          <label
            class="chakra-checkbox css-135hl63"
          >
            <input
              class="chakra-checkbox__input"
              id="isPeriodDate"
              name="isPeriodDate"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="checkbox"
              value=""
            />
            <span
              aria-hidden="true"
              class="chakra-checkbox__control css-dnty2r"
            />
          </label>
          <p
            class="chakra-text css-1rvolpk"
          >
            I want this to be a period of time
          </p>
        </div>
        <div
          class="css-1n4fp72"
        >
          <div
            class="css-lo4xcv"
          >
            Enter
             Date
            <span
              class="chakra-text css-1b8yx6g"
            >
               
              *
            </span>
          </div>
          <div
            class="css-iyc5oy"
          >
            <div
              class="css-17eahu8"
            >
              <div
                class=" css-n9lnso-SelectContainer"
              >
                <span
                  class="css-1f43avz-a11yText-A11yText"
                  id="react-select-:r0:-live-region"
                />
                <span
                  aria-atomic="false"
                  aria-live="polite"
                  aria-relevant="additions text"
                  class="css-1f43avz-a11yText-A11yText"
                />
                <div
                  class=" css-6o1t9b-control"
                >
                  <div
                    class=" css-15ki4p9-ValueContainer"
                  >
                    <div
                      class="css-xxdqwu"
                    >
                      <div
                        class="css-1yks822"
                      >
                        DD
                      </div>
                      <div
                        class=" css-qbdosj-Input"
                        data-value=""
                      >
                        <input
                          aria-autocomplete="list"
                          aria-describedby="react-select-:r0:-placeholder"
                          aria-expanded="false"
                          aria-haspopup="true"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          class=""
                          id="react-select-:r0:-input"
                          role="combobox"
                          spellcheck="false"
                          style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class=" css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      class=" css-1xjgjl1-IndicatorSeparator"
                    />
                    <svg
                      class="chakra-icon css-fgc0y"
                      focusable="false"
                      viewBox="0 0 20 20"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.25 6.875L10 13.125L3.75 6.875"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.5"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="css-17eahu8"
            >
              <div
                class=" css-n9lnso-SelectContainer"
              >
                <span
                  class="css-1f43avz-a11yText-A11yText"
                  id="react-select-:r1:-live-region"
                />
                <span
                  aria-atomic="false"
                  aria-live="polite"
                  aria-relevant="additions text"
                  class="css-1f43avz-a11yText-A11yText"
                />
                <div
                  class=" css-6o1t9b-control"
                >
                  <div
                    class=" css-15ki4p9-ValueContainer"
                  >
                    <div
                      class="css-xxdqwu"
                    >
                      <div
                        class="css-1yks822"
                      >
                        MM
                      </div>
                      <div
                        class=" css-qbdosj-Input"
                        data-value=""
                      >
                        <input
                          aria-autocomplete="list"
                          aria-describedby="react-select-:r1:-placeholder"
                          aria-expanded="false"
                          aria-haspopup="true"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          class=""
                          id="react-select-:r1:-input"
                          role="combobox"
                          spellcheck="false"
                          style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class=" css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      class=" css-1xjgjl1-IndicatorSeparator"
                    />
                    <svg
                      class="chakra-icon css-fgc0y"
                      focusable="false"
                      viewBox="0 0 20 20"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.25 6.875L10 13.125L3.75 6.875"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.5"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="css-17eahu8"
            >
              <div
                class=" css-n9lnso-SelectContainer"
              >
                <span
                  class="css-1f43avz-a11yText-A11yText"
                  id="react-select-:r2:-live-region"
                />
                <span
                  aria-atomic="false"
                  aria-live="polite"
                  aria-relevant="additions text"
                  class="css-1f43avz-a11yText-A11yText"
                />
                <div
                  class=" css-6o1t9b-control"
                >
                  <div
                    class=" css-15ki4p9-ValueContainer"
                  >
                    <div
                      class="css-xxdqwu"
                    >
                      <div
                        class="css-1yks822"
                      >
                        YYYY
                      </div>
                      <div
                        class=" css-qbdosj-Input"
                        data-value=""
                      >
                        <input
                          aria-autocomplete="list"
                          aria-describedby="react-select-:r2:-placeholder"
                          aria-expanded="false"
                          aria-haspopup="true"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          class=""
                          id="react-select-:r2:-input"
                          role="combobox"
                          spellcheck="false"
                          style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class=" css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      class=" css-1xjgjl1-IndicatorSeparator"
                    />
                    <svg
                      class="chakra-icon css-fgc0y"
                      focusable="false"
                      viewBox="0 0 20 20"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M16.25 6.875L10 13.125L3.75 6.875"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.5"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-omcqsq"
        >
          <div
            class="css-1xnzs7r"
          >
            Enter Name/Title
            <span
              class="chakra-text css-1b8yx6g"
            >
               
              *
            </span>
          </div>
          <textarea
            class="chakra-textarea css-pkraa3"
            id="title"
            name="title"
            placeholder="It can be name of Medal/Certificate/Prize or anything that important to you"
            style="padding-left: 0px; border-radius: 0;"
          />
        </div>
        <div
          class="css-omcqsq"
        >
          <div
            class="css-1xnzs7r"
          >
            Enter Description/Subtitle
          </div>
          <textarea
            class="chakra-textarea css-pkraa3"
            id="description"
            name="description"
            placeholder="Describe further for your fan to understand (Where did it happen/ How...)"
            style="padding-left: 0px; border-radius: 0;"
          />
        </div>
        <div
          class="css-19idom"
        >
          <div
            class="css-qhhr1q"
          >
            Choose an icon to represent the milestone
          </div>
          <div
            class="css-cewebh"
          >
            Make your milestone even more interesting!
          </div>
          <div
            class="chakra-stack css-1689b8b"
          >
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-1y5pyw7"
                focusable="false"
                viewBox="0 0 16 28"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12.8516 0.863892C12.5586 2.00706 12.2715 3.13088 11.9785 4.24124C9.32422 4.66721 6.67578 4.6678 4.02148 4.24124C3.72852 3.13088 3.44141 2.00706 3.14844 0.863892C6.36523 2.18694 9.63476 2.18694 12.8516 0.863892ZM13.8652 1.13284L15.8809 3.82522C15.0605 9.46487 13.5371 15.7871 11.3398 21.5762C10.7129 20.8496 9.86328 20.3282 8.89063 20.1289C10.5488 13.6074 12.207 7.68948 13.8652 1.13284ZM2.13477 1.13284C3.79297 7.68948 5.45117 13.6074 7.10938 20.1289C6.13672 20.3282 5.28711 20.8496 4.66016 21.5762C2.46289 15.7871 0.939453 9.46487 0.119141 3.82522L2.13477 1.13284ZM8 21.0899C9.85156 21.0899 11.3398 22.5782 11.3398 24.4297C11.3398 26.2813 9.85156 27.7696 8 27.7696C6.14844 27.7696 4.66016 26.2813 4.66016 24.4297C4.66016 22.5782 6.14844 21.0899 8 21.0899Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-1617iy1"
                focusable="false"
                viewBox="0 0 26 26"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16.7992 13.9875L6.17422 3.3625C5.93672 3.125 5.61172 3 5.28672 3C4.96172 3 4.64922 3.125 4.41172 3.3625L0.861719 6.9125C0.374219 7.4 0.374219 8.2 0.861719 8.6875L11.4867 19.3125C11.7367 19.5625 12.0492 19.675 12.3742 19.675C12.6992 19.675 13.0117 19.55 13.2617 19.3125L16.7992 15.775C17.2867 15.275 17.2867 14.475 16.7992 13.9875V13.9875ZM12.3742 16.65L3.51172 7.7875L5.28672 6.0125L14.1492 14.875L12.3742 16.65V16.65ZM15.9255 20.195L17.693 18.4275L22.9967 23.7312L21.2292 25.4988L15.9255 20.195ZM21.1242 0.5C18.7117 0.5 16.7492 2.4625 16.7492 4.875C16.7492 7.2875 18.7117 9.25 21.1242 9.25C23.5367 9.25 25.4992 7.2875 25.4992 4.875C25.4992 2.4625 23.5367 0.5 21.1242 0.5ZM21.1242 6.75C20.0867 6.75 19.2492 5.9125 19.2492 4.875C19.2492 3.8375 20.0867 3 21.1242 3C22.1617 3 22.9992 3.8375 22.9992 4.875C22.9992 5.9125 22.1617 6.75 21.1242 6.75Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-12t7xd"
                focusable="false"
                viewBox="0 0 18 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M7.75 2.5H5.25V0H7.75V2.5ZM12.75 0H10.25V2.5H12.75V0ZM5.25 12.5H7.75V10H5.25V12.5ZM17.75 7.5V5H15.25V7.5H17.75ZM17.75 12.5V10H15.25V12.5H17.75ZM10.25 12.5H12.75V10H10.25V12.5ZM17.75 0H15.25V2.5H17.75V0ZM10.25 5V2.5H7.75V5H10.25ZM2.75 7.5V5H5.25V2.5H2.75V0H0.25V20H2.75V10H5.25V7.5H2.75ZM12.75 10H15.25V7.5H12.75V10ZM7.75 7.5V10H10.25V7.5H7.75ZM5.25 5V7.5H7.75V5H5.25ZM10.25 7.5H12.75V5H10.25V7.5ZM12.75 2.5V5H15.25V2.5H12.75Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-7wjvle"
                focusable="false"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M10 13.3333C13.225 13.3333 15.8334 10.725 15.8334 7.49996C15.8334 4.27496 13.225 1.66663 10 1.66663C6.77502 1.66663 4.16669 4.27496 4.16669 7.49996C4.16669 10.725 6.77502 13.3333 10 13.3333ZM10 3.33329C12.3 3.33329 14.1667 5.19996 14.1667 7.49996C14.1667 9.79996 12.3 11.6666 10 11.6666C7.70002 11.6666 5.83335 9.79996 5.83335 7.49996C5.83335 5.19996 7.70002 3.33329 10 3.33329Z"
                  fill="currentColor"
                />
                <path
                  d="M8.33333 7.50004C8.79357 7.50004 9.16667 7.12694 9.16667 6.66671C9.16667 6.20647 8.79357 5.83337 8.33333 5.83337C7.8731 5.83337 7.5 6.20647 7.5 6.66671C7.5 7.12694 7.8731 7.50004 8.33333 7.50004Z"
                  fill="currentColor"
                />
                <path
                  d="M11.6666 7.50004C12.1269 7.50004 12.5 7.12694 12.5 6.66671C12.5 6.20647 12.1269 5.83337 11.6666 5.83337C11.2064 5.83337 10.8333 6.20647 10.8333 6.66671C10.8333 7.12694 11.2064 7.50004 11.6666 7.50004Z"
                  fill="currentColor"
                />
                <path
                  d="M10 5.83329C10.4603 5.83329 10.8334 5.4602 10.8334 4.99996C10.8334 4.53972 10.4603 4.16663 10 4.16663C9.53978 4.16663 9.16669 4.53972 9.16669 4.99996C9.16669 5.4602 9.53978 5.83329 10 5.83329Z"
                  fill="currentColor"
                />
                <path
                  d="M5.83331 15.8333H7.49998C8.41665 15.8333 9.16665 16.5833 9.16665 17.5V18.3333H10.8333V17.5C10.8333 16.5833 11.5833 15.8333 12.5 15.8333H14.1666V14.1666H5.83331V15.8333Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-1f4sfca"
                focusable="false"
                viewBox="0 0 26 28"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M22.4008 2.11251C19.4758 -0.812492 14.1258 -0.224992 10.4633 3.43751C8.46328 5.43751 7.31328 8.27501 7.28828 10.2625C7.26328 12.2375 7.61328 15.125 5.60078 17.1375L0.300781 22.4375L2.07578 24.2125L7.37578 18.9125C9.38828 16.9 12.2758 17.25 14.2508 17.225C16.2258 17.2 19.0758 16.05 21.0758 14.05C24.7258 10.3875 25.3258 5.03751 22.4008 2.11251ZM10.9008 13.6C8.98828 11.6875 9.58828 7.83751 12.2258 5.20001C14.8633 2.56251 18.7008 1.96251 20.6258 3.87501C22.5383 5.78751 21.9383 9.63751 19.3008 12.275C16.6633 14.9125 12.8258 15.5125 10.9008 13.6ZM20.5008 20.25C21.1633 20.25 21.8008 20.5125 22.2633 20.9875C23.2383 21.9625 23.2383 23.55 22.2633 24.525C21.8008 24.9875 21.1633 25.25 20.5008 25.25C19.8383 25.25 19.2008 24.9875 18.7383 24.5125C17.7633 23.5375 17.7633 21.95 18.7383 20.975C19.2008 20.5125 19.8383 20.25 20.5008 20.25ZM20.5008 17.75C19.5108 17.748 18.5425 18.0401 17.7187 18.5893C16.895 19.1384 16.2529 19.9199 15.8739 20.8345C15.495 21.7491 15.3962 22.7557 15.5901 23.7265C15.7841 24.6973 16.262 25.5887 16.9633 26.2875C17.9383 27.2625 19.2258 27.75 20.5008 27.75C21.4908 27.752 22.4591 27.4599 23.2828 26.9107C24.1066 26.3616 24.7487 25.5801 25.1276 24.6655C25.5066 23.7509 25.6054 22.7443 25.4114 21.7735C25.2175 20.8027 24.7396 19.9113 24.0383 19.2125C23.5737 18.7482 23.0221 18.3801 22.4151 18.1291C21.8081 17.8782 21.1576 17.7494 20.5008 17.75Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <button
              class="chakra-button css-n4jjqw"
              type="button"
            >
              <svg
                class="chakra-icon css-1617iy1"
                focusable="false"
                viewBox="0 0 26 26"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M13 0.5C6.1 0.5 0.5 6.1 0.5 13C0.5 19.9 6.1 25.5 13 25.5C19.9 25.5 25.5 19.9 25.5 13C25.5 6.1 19.9 0.5 13 0.5ZM14.25 4.625L15.9375 3.4375C18.1987 4.13819 20.1384 5.61737 21.4125 7.6125L20.925 9.2875L19.2375 9.8625L14.25 6.375V4.625ZM10.0625 3.4375L11.75 4.625V6.375L6.7625 9.8625L5.075 9.2875L4.5875 7.6125C5.86899 5.62421 7.80597 4.14715 10.0625 3.4375ZM6.85 19.3875L5.425 19.5125C3.85806 17.705 2.99687 15.3922 3 13C3 12.85 3.0125 12.7125 3.025 12.5625L4.275 11.65L6 12.25L7.825 17.675L6.85 19.3875ZM16.125 22.4875C15.1375 22.8125 14.0875 23 13 23C11.9125 23 10.8625 22.8125 9.875 22.4875L9.0125 20.625L9.8125 19.25H16.2L17 20.6375L16.125 22.4875ZM15.8375 16.75H10.1625L8.475 11.725L13 8.55L17.5375 11.725L15.8375 16.75ZM20.575 19.5125L19.15 19.3875L18.1625 17.675L19.9875 12.25L21.725 11.6625L22.975 12.575C22.9875 12.7125 23 12.85 23 13C23 15.4875 22.0875 17.7625 20.575 19.5125Z"
                  fill="currentColor"
                />
              </svg>
            </button>
          </div>
        </div>
        <button
          class="chakra-button css-ydjbwb"
          type="submit"
        >
          SAVE
        </button>
      </form>
    </div>
  </div>
</div>
`;
